cfme/tests/cloud_infra_common/test_html5_vm_console.py

```diff
@@ -171,9 +171,15 @@ def _validate_login():
             time.sleep(15)
 
         # create file on system
-        vm_console.send_keys("touch blather\n")
-        logger.info("Wait 5 sec for file creation")
-        time.sleep(5)  # Wait for file creation
+        def _touch_file():
+            vm_console.send_keys("\n\ntouch blather\n")
+            logger.info("Wait 5 sec for file creation")
+            time.sleep(5)  # Wait for file creation
+            logger.info("vm_console.find_text_on_screen(text_to_find='touch blather') is True {}"
+                .format(vm_console.find_text_on_screen(text_to_find='touch blather') is True))
+            return vm_console.find_text_on_screen(text_to_find='touch blather') is True
+
+        wait_for(func=_touch_file, timeout=180, delay=5)
```

Comment:
you're waiting 5s for the file creation within `_touch_file`, I would drop this delay to .1